### PR TITLE
Ft/navbar

### PR DESCRIPTION
**Pull request by Nezerwa**
# Description
In this Pr we : 

- [ ] Created our header component.
- [ ] Added Ojemba logo
- [ ] Added Navigation bar
- [ ] Rendered our header component in index.html

## Motivation and Context
-The changes in this Pr create and render A header section that includes a logo and Navigation bar in the index.html

## Screenshots

changedheader

### CODE REVIEW

File: assets/images/logo.svg
Comment (peninah98) [inline assets/images/logo.svg:1]: The logo is okaz!

**Review by ell-sie — CHANGES_REQUESTED** [review]
Great job on the page clone, team! Let's ensure that we stick to replicating the original design without adding any elements, such as the language selection dropdown in the header, that weren't present on the original page.

**Review by peninah98 — CHANGES_REQUESTED** [review]
Change the dropdown to EN, follow figma file instructions here is the link [figma design]( https://www.figma.com/file/YRF8ZXuCSDqwmBI9iurvVC/Ojemba.de-VLM-(Copy)?type=design&node-id=0%3A1&mode=design&t=rPH3Qzjc4b42DYhe-1 
)

**Review by peninah98 — APPROVED** [review]

**Review by ell-sie — APPROVED** [review]
Well done everything looks on point!

**Pull request merged**